engine/src/main/java/org/destinationsol/game/GalaxyFiller.java

```diff
@@ -120,6 +122,15 @@ private FarShip build(SolGame game, ShipConfig cfg, Faction faction, boolean mai
         }
         return s;
     }
+    
+    public JsonValue checkRootNull(Json json) {
```

Comment:
CheckRootNull implies you are just checking, and should return a boolean. Since our primary objective is not checking but rather obtatining the JsonValue, this should be names slightly differently. `GetRootNode()` ?